portal-web/test/functional/com/liferay/portalweb/macros/layout/displaypagetemplate/DisplayPageTemplate.macro

```diff
@@ -34,6 +34,15 @@ definition {
 						key_card = "${assetTitle}",
 						locator1 = "Card#CARD_TITLE");
 				}
+				else if ("${assetType}" == "Category") {
+					Portlet.expandTree();
+
+					Pause(locator1 = "3000");
```

Comment:
@Tim-Cao are we sure we can not avoid this? can we request something from Poshi team to do this in a better way?